test/models/user_test.rb

```diff
@@ -0,0 +1,56 @@
+require "test_helper"
+
+describe User do
+  describe "validations" do
+    it "can be instantiated" do
+   
+      expect users(:user_1).valid?.must_equal true
```

Comment:
You should use parenthesis with `expect`:

```suggestion
      expect(users(:user_1).valid?).must_equal true
```